Tests/SWBCoreTests/SettingsTests.swift

```diff
@@ -3003,7 +3003,7 @@ import SWBMacro
                                 "ENABLE_XOJIT_PREVIEWS": "YES",
                                 "CODE_SIGN_IDENTITY": "An Engineer",
 
-                                "ENABLE_HARDED_RUNTIME_EXPECTED": "YES",
```

Comment:
Is this an indication that the test is incorrect, since it didn’t fail with an invalid flag?

Reply:
The name of the build setting in this case was arbitrary and didn't require matching up with another spelling elsewhere.